linuxNew/jdk/suse/src/main/packaging/Dockerfile

```diff
@@ -25,38 +31,11 @@ RUN --mount=type=secret,id=gpg,gid=10001,uid=10000,dst=/tmp/private.gpg \
 '\
 >> /home/builder/.rpmmacros; \
 	fi
-USER root
 
 # Prepare entrypoint and build scripts
 ADD entrypoint.sh /entrypoint.sh
-RUN printf '#!/usr/bin/env bash\n\
-set -euxo pipefail\n\
-\n\
-# Ensure necessary directories for rpmbuild operation are present.\n\
-rpmdev-setuptree \n\
-\n\
-# Build all spec files we can find.\n\
-
-targets="x86_64 ppc64le s390x aarch64 armv7hl" \n\
-for spec in "$(ls /home/builder/build/generated/packaging/*.spec)"; do\n\
-	rpmdev-spectool -g -R "$spec";\n\
-	rpmbuild --nodeps -bs "$spec"; \n\
-	if [[ "$spec" =~ "temurin-8-jdk" ]]; then \n\
-		targets="x86_64 ppc64le aarch64 armv7hl" \n\
-	fi \n\
-	for target in $targets; do \n\
-		rpmbuild --target "$target" --rebuild /home/builder/rpmbuild/SRPMS/*.src.rpm;\n\
-	done;\n\
-done;\n\
-\n\
-# Copy generated RPMs to destination folder\n\
-find /home/builder/rpmbuild/SRPMS /home/builder/rpmbuild/RPMS -type f -name "*.rpm" -print0 | xargs -0 -I {} cp {} /home/builder/out\n\
-# Sign generated RPMs with rpmsign.\n\
-if grep -q %%_gpg_name /home/builder/.rpmmacros; then \n\
-	rpmsign --addsign /home/builder/out/*.rpm \n\
-fi;\n\
-'\
->> /home/builder/build.sh
+ADD build.sh /home/builder/build.sh
+USER root
```

Comment:
Same comment as to why this has been added

Reply:
same as above, not added by this PR

Reply:
Ah I've just seen that it's just been moved to a different place in the file. So it's now after the addition of the entrypoint and build.sh files and no longer owned by the `builder` ID